### PR TITLE
[step 1] Missing fields transactions

### PR DESCRIPTION
**Pull request by eesmerdag**
first step fetching with current info

https://www.notion.so/figmentnetworks/Include-Transaction-Fee-for-Near-Indexer-transaction-response-76d857768e02433db3a189f99a7c5cbd

### CODE REVIEW

File: near/types.go [hunk 1]
```diff
@@ -150,6 +150,7 @@ type Validator struct {
 type ReceiptsOutcome struct {
 	BlockHash string  `json:"block_hash"`
 	ID        string  `json:"id"`
+	Proof     []Proof `json:"proof"`
```
[the remark this answers is not in the document]
Comment (eesmerdag) [reply near/types.go:153]: **/transactions** will be seen in this endpoint response

File: near/types.go [hunk 2]
```diff
@@ -165,18 +166,26 @@ type ActionError struct {
...
+	ExecutorId  string        `json:"executor_id"`
+	Logs        []interface{} `json:"logs"`
+	ReceiptIds  []string      `json:"receipt_ids"`
+	Status      Status        `json:"status"`
 }
 
 type TransactionOutcome struct {
 	BlockHash string  `json:"block_hash"`
 	ID        string  `json:"id"`
+	Proof     []Proof `json:"proof"`
```
[the remark this answers is not in the document]
Comment (eesmerdag) [reply near/types.go:180]: **/transactions** will be seen in this endpoint response

File: model/mapper/transaction.go [hunk 3]
```diff
@@ -22,10 +22,29 @@ func Transaction(block *near.Block, input *near.TransactionDetails) (*model.Tran
 		Time:      util.ParseTime(block.Header.Timestamp),
 		Sender:    tx.SignerID,
 		Receiver:  tx.ReceiverID,
-		Amount:    types.NewAmount("0"),
+		PublicKey: tx.PublicKey,
```
[the remark this answers is not in the document]
Comment (eesmerdag) [reply model/mapper/transaction.go:25]: I knew we wanted to save whatever we could save and it is the part of the same response https://docs.near.org/docs/api/rpc#transaction-status So I made it a part of the code. we can remove it if you want.

File: model/util/calculate.go [hunk 4]
```diff
@@ -11,3 +18,23 @@ func Percentage(max int, cur int) float64 {
...
 }
+
+// CalculateTransactionFee calculates transaction fee
+func CalculateTransactionFee(trx near.TransactionDetails) (string, error) {
+	fee := new(big.Int)
+	o := new(big.Int)
+	r := new(big.Int)
+	_, ok := o.SetString(trx.TransactionOutcome.Outcome.TokensBurnt, 10)
+	if !ok {
+		return "", errors.New("error with tokens burnt field")
```
[the remark this answers is not in the document]
Comment (eesmerdag) [reply model/util/calculate.go:29]: 👍 i will change it

File: model/mapper/transaction.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (sosedoff) [reply model/mapper/transaction.go:27]: pubkey/sig is nice to have, so we'll keep it around

File: cli/status.go [hunk 5]
```diff
@@ -23,9 +22,6 @@ func startStatus(cfg *config.Config) error {
 	defer db.Close()
 
 	rpcEndpoints := strings.Split(cfg.RPCEndpoints, ",")
-	if len(rpcEndpoints) != 1 {
```
Comment (pacmessica) [inline cli/status.go:26]: 👍

**Review by pacmessica — DISMISSED** [review]
lgtm

File: near/types.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (pacmessica) [reply near/types.go:180]: isn't that for Outcome Outcome json:"outcome"``?

We're not saving Proof to db, did you mean to?

[the remark this answers is not in the document]
Comment (sosedoff) [reply near/types.go:180]: Proof is included in multiple objects, but i dont think we need to store it, it does not have any relevant information.

**Review by pacmessica — APPROVED** [review]

**Pull request merged**